spec/rails_helper.rb

```diff
@@ -30,6 +30,8 @@
 Dir[Rails.root.join('..', 'spec', 'support', '**', '*.rb')]
   .sort.each { |f| require f }
 
+Devise.stretches = 1
+Rails.logger.level = 4
```

Comment:
I'm not sure why we need this... can you add a small comment detailing why this was added here?

Reply:
Of course I can.